core/src/main/scala/org/http4s/MediaType.scala

```diff
@@ -260,6 +260,7 @@ object MediaType extends Registry {
   val `application/postscript` = app("postscript", compressible, binary, "ai", "eps", "ps")
   val `application/rss+xml` = app("rss+xml", compressible, notBinary, "rss")
   val `application/soap+xml` = app("soap+xml", compressible, notBinary)
+  val `application/vnd.api+json` = app("vnd.api+json", compressible, binary) // we treat JSON as binary, since it's encoding is not variable but defined by RFC4627
```

Comment:
Does this need to be added to the types in logger that actually gets parsed as text despite being binary?

Reply:
Not really sure. Just doing it the same as the other json media types.